arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -1436,6 +1436,10 @@ void RocksDBEngine::trackRevisionTreeBufferedMemoryDecrease(
   TRI_ASSERT(old >= value);
 }
 
+uint64_t RocksDBEngine::treeBufferedMemoryUsage() const noexcept {
+  return _metricsTreeBufferedMemoryUsage.load();
+}
+
```

Comment:
I added this only for testing. Currently not called from anywhere, so can be removed again if required.
```suggestion
```